target_api/target.py

```diff
@@ -39,7 +39,6 @@ def __init__(
         state: str = None
     ) -> None:
         super().__init__(config, parse_env_config, validate_config, state)
-        self.streaming_job = os.environ.get("STREAMING_JOB")
```

Comment:
Why did you remove this? Isn't it being used in other places?

Reply:
It is still being used, but since this class subclasses `TargetHotglue` and calls its initializer, it is redundant since it is also defined [there](https://gitlab.com/hotglue/target-hotglue-sdk/-/merge_requests/23).